lib/vapi_cli/trained-models.py

```diff
@@ -207,14 +208,15 @@ def deploy(params):
 
     model = params.get("--modelid", "missing_id")
     name = params.get("--name", None)
+    datasetid = params.get("--datasetid", None)
```

Comment:
This should be added to the `expectedArgs`. That way, you do not have to change the signature of the library.  We must preserve that API signature so that anybody using the library directly (e.g. automation test scripts) do not also have to be change because there is a new parameter.
Only required parameters should be passed directly, all other should be in the `kwargs`.